assets/app/scripts/filters/util.js

```diff
@@ -341,4 +344,39 @@ angular.module('openshiftConsole')
     return function(resource, kind, namespace) {
       return Navigate.resourceURL(resource, kind, namespace);
     };
+  })
+  .filter('join', function() {
+    return function(array, separator) {
+      if (!separator) {
+        separator = ',';
+      }
+      return array.join(separator);
+    };
+  })
+  .filter('accessModes', function() {
+    return function(value, format) {
+      if (!value) {
+        return value;
+      }  
+      var accessModes = [];
+      angular.forEach(value, function(item) {
+        var accessModeString;
+        var longForm = format === "long";
+        switch(item) {
+          case "ReadWriteOnce":
+            accessModeString =  longForm ? "RWO (Read-Write-Once)" : "Read-Write-Once";
+            break;
+          case "ReadOnlyMany":
+            accessModeString = longForm ? "ROX (Read-Only-Many)" : "Read-Only-Many";
+            break;
+          case "ReadWriteMany":
+            accessModeString = longForm ? "RWX (Read-Write-Many)" : "Read-Write-Many";
+            break;
+        }
```

Comment:
I'll actually change to use the value as-is if it's not one of the ones we recognize.